openapi/platform/platform.yaml

```diff
@@ -6899,11 +6879,16 @@ components:
           example: proposals
           enum:
             - proposals
+          readOnly: true
         id:
           type: string
           description: ignored when creating a proposal
+          readOnly: true
         relationships:
           type: object
+          required:
+            - creatorOrganization
+            - creatorUser
```

Comment:
Unfortunately, because the model is shared by the create and update requests you won't be able to make this change. `type` is required on both. `id` is required on update. All other fields are optional on updates. (I'm looking into using codegen to allow us to split the models without needing a bunch of manual work to keep them in sync.)